### DISCUSSION TITLE
Tailwind script from cdn minus styles affecting base elements like body and html

### DISCUSSION THREAD

**Discussion by jwilkeson** [tailwindlabs/tailwindcss#8640] [Help]
Is there a script I can use perhaps from a cdn that has the tailwind classes but doesn't affect anything that doesn't have those classes? By that I mean it doesn't change the styles of the body, html or any other base elements.

**Comment by rohmann** [comment]
@jwilkeson The boilerplate styles are included via the core `preflight` plugin. That can be disabled through the configuration, but you could find yourself in a bunch of different peculiar situation where things missing normalization styles don't look quite right. You'll just need to keep that in mind and account for it.

The Tailwind CDN can be configured via `tailwind.config` (https://tailwindcss.com/docs/installation/play-cdn) So if you use the code below, you can pull in the CDN with `preflight` disabled. Just remember it's not meant for production.

```html
<script src="https://cdn.tailwindcss.com"></script>
<script>
  tailwind.config = {
    corePlugins: {
      preflight: false
    }
}
</script>
```

**Reply by LoganDark** [reply]
thank you so much! this worked for me
